truffle/com.oracle.truffle.sl.test/src/com/oracle/truffle/sl/test/SLTckTest.java

```diff
@@ -212,6 +213,74 @@ protected String evaluateSource() {
         return "eval";
     }
 
+    //
+    // Ignore tests working on floats and double
+    //
+
```

Comment:
Now there is "assertDouble" method in the TCK overriden by SLTckTest to do nothing. That should be enough to let the test run, but not check the output value. Please remove all these overrides and make sure your tests in TruffleTCK use assertDouble.